src/Console/DuskCommand.php

```diff
@@ -45,13 +46,29 @@ public function handle()
 
         $options = implode(' ', array_slice($_SERVER['argv'], 2));
 
-        $this->withDuskEnvironment(function () use ($options) {
-            (new Process(trim('php vendor/bin/phpunit -c "'.base_path('phpunit.dusk.xml').'" '.$options), base_path(), []))
-                    ->setTty(true)
-                    ->run(function ($type, $line) {
-                        $this->output->write($line);
-                    });
-        });
+        $this->withDuskEnvironment(
+            function () use ($options) {
+                (new ProcessBuilder())
+                    ->setPrefix($this->getPathToPhpUnit())
+                    ->setArguments(['-c', base_path('phpunit.dusk.xml'), $options])
+                    ->getProcess()
+                    ->setTty(PHP_OS !== 'WINNT')
+                    ->run(
+                        function ($type, $line) {
+                            $this->output->write($line);
+                        }
+                    );
+            }
+        );
+    }
+
+    private function getPathToPhpUnit()
+    {
+        if (PHP_OS === 'WINNT') {
+            return base_path('vendor\bin\phpunit.bat');
+        }
+
+        return 'php vendor/bin/phpunit';
```

Comment:
It was from first version. I tried to keep all that is not connected with windows unchanged. If you have linux or macOS can you check if it work with `base_path()`?